### PR TITLE
Rewrite formulas using mathjax

### LINKED ISSUE #355: Rewrite numerical formulas in documentation by using MathJax.
To improve readability, we would like to rewrite numerical formulas in documentation by using MathJax.

### PR DESCRIPTION
**Pull request by crcrpar**
Closes #355.

### CODE REVIEW

File: optuna/trial.py [hunk 1]
```diff
@@ -187,10 +189,13 @@ def suggest_discrete_uniform(self, name, low, high, q):
...
-        ``q``. More specifically, this method returns one of the values in the sequence ``low,
-        low + q, low + 2 * q, ..., low + k * q <= high``, where `k`` denotes an integer. Note that
-        ``high`` may be excluded from ranges due to round-off errors if ``q`` is not an integer.
+        The value is sampled from the range :math:`[\\mathsf{low}, \\mathsf{high}]`,
+        and the step of discretization is :math:`q`. More specifically,
+        this method returns one of the values in the sequence
+        :math:`\\mathsf{low}, \\mathsf{low} + q, \\mathsf{low} + 2 q, \\dots,
+        \\mathsf{low} + k q \\le \\mathsf{high}`,
+        where :math:`k` denotes an integer. Note that :math:`high` may be
+        excluded from ranges due to round-off errors if ``q`` is not an integer.
```
Comment (sile) [inline optuna/trial.py]: I think it is better to replace ``` ``q`` ``` with ``` :math:`q` ```.

Comment (crcrpar) [reply optuna/trial.py]: Thanks.

**Comment by toshihikoyanase** [comment]
@crcrpar
Thank you for your contribution!

**Comment by crcrpar** [comment]
Awesome, the pleasure is all mine.

**Pull request merged**